teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/DebugDbCommand.java

```diff
@@ -407,325 +402,29 @@ public int dumpBlobsSidecar(
                 Files.newOutputStream(
                     outputFile, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING))) {
 
-      try (final Stream<Map.Entry<SlotAndBlockRoot, Bytes>> streamBlobsSidecarAsSsz =
-          database.streamBlobsSidecarsAsSsz(from, to)) {
-        for (final Iterator<Map.Entry<SlotAndBlockRoot, Bytes>> iterator =
-                streamBlobsSidecarAsSsz.iterator();
-            iterator.hasNext(); ) {
-          final Map.Entry<SlotAndBlockRoot, Bytes> entryData = iterator.next();
+      try (final Stream<SlotAndBlockRootAndBlobIndex> keyStream =
+          database.streamBlobSidecarKeys(from, to)) {
+
+        for (final Iterator<SlotAndBlockRootAndBlobIndex> it = keyStream.iterator();
+            it.hasNext(); ) {
+          final SlotAndBlockRootAndBlobIndex key = it.next();
+          if (key.isNoBlobsKey()) {
+            continue;
+          }
+          final BlobSidecar blobSidecar = database.getBlobSidecar(key).orElseThrow();
           out.putNextEntry(
               new ZipEntry(
-                  entryData.getKey().getSlot()
+                  blobSidecar.getSlot()
                       + "_"
-                      + entryData.getKey().getBlockRoot().toUnprefixedHexString()
+                      + blobSidecar.getBlockRoot().toUnprefixedHexString()
                       + ".ssz"));
-          out.write(entryData.getValue().toArrayUnsafe());
+          blobSidecar.sszSerialize(out);
           index++;
         }
       }
 
-      System.out.println("Wrote " + index + " blobs sidecars to " + outputFile.toAbsolutePath());
-
-      index = 0;
-      try (final Stream<SlotAndBlockRoot> unconfirmedBlobs =
-          database.streamUnconfirmedBlobsSidecars(from, to)) {
-        out.putNextEntry(new ZipEntry("unconfirmed.txt"));
-        for (final Iterator<SlotAndBlockRoot> iterator = unconfirmedBlobs.iterator();
-            iterator.hasNext(); ) {
-          final SlotAndBlockRoot entryData = iterator.next();
-          out.write(entryData.toLogString().getBytes(StandardCharsets.UTF_8));
-          if (iterator.hasNext()) {
-            out.write("\n".getBytes(StandardCharsets.UTF_8));
-          }
-          index++;
-        }
-      }
-    }
-
-    System.out.println(
-        "Wrote " + index + " unconfirmed blobs sidecars to " + outputFile.toAbsolutePath());
-    return 0;
-  }
-
-  @Command(
-      name = "validate-blobs-sidecars",
```

Comment:
Why you completely removed this command? It was useful to me to check that we have all blobs we expect to have

Reply:
We'd better to start it from the blank, there are soo many changes in categories